tests/test_location.py

```diff
@@ -106,7 +140,6 @@ def testRelativeRoot(self):
 
         pathInStore = "relative/path/file.ext"
         loc1 = factory.fromPath(pathInStore)
-
```

Comment:
Can you put this line back (and squash it) to minimize diff? Removing the line doesn't give us anything.

Reply:
I don't know where they keep going, these empty lines that is. Thanks!